pkg/metricshandler/metrics_handler.go

```diff
@@ -187,6 +187,9 @@ func (m *MetricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	contentType := expfmt.NegotiateIncludingOpenMetrics(r.Header)
 
+	if contentType != expfmt.FmtOpenMetrics {
```

Comment:
```suggestion
        # We do not support protobuf at the moment. Fall back to FmtText if the negotiated exposition format is not FmtOpenMetrics See: https://github.com/kubernetes/kube-state-metrics/issues/2022
	if contentType != expfmt.FmtOpenMetrics {
```

Add a comment so it's clear why we're doing it.